exercises/react/example.cr

```diff
@@ -0,0 +1,71 @@
+alias CellValue = Int32
+
+private class Cell
+  @dependents = [] of ComputeCell
+
+  def initialize(initial : CellValue)
+    @value = initial
+  end
+
+  getter :value
+
+  protected getter :dependents
+  protected def propagate; _propagate; end
```

Comment:
Actually... after some though, I don't think this can be solved, and the behaviour is probably correct because you are effectively invoking protected methods in unrelated types. Ruby also gives an error for this, so... Nevermind :-)